run-tests.sh

```diff
@@ -94,6 +94,7 @@ if [ $# -eq 0 ]
         info_msg "Sphinx-build:"
         pipenv run sphinx-build -qnNW docs docs/_build/html
         info_msg "Tests:"
+        # pipenv run invenio utils set_test_static_folder -v
```

Comment:
Why not using the CLI you created before? It's like a "test" to launch it in test, isn't it?